### ISSUE TITLE
Get working on Windows

### ISSUE THREAD

**Issue by mstrimas** [mstrimas/auk#2]

**Comment by mstrimas** [comment]
Potentially useful links:
https://stackoverflow.com/questions/5068264/cant-call-cygwin-commands-from-r
https://stackoverflow.com/questions/20315824/how-to-call-cygwin-command-from-r

**Comment by mstrimas** [comment]
Possible way to check if awk is installed on Windows:
`system("C:/cygwin64/bin/gawk.exe --version")`
`system("C:/cygwin/bin/gawk.exe --version")`

**Comment by mstrimas** [comment]
Calling gawk via Windows:
```
system.file("extdata/zerofill-ex_ebd.txt", package = "auk") %>% 
  sprintf("C:/cygwin64/bin/gawk.exe '{print $1}' %s", .) %>% 
  system()
```

**Comment by mstrimas** [comment]
sample data https://drive.google.com/open?id=0B7qaDpMkO09uS3NZY3RmTHpSS28

**Comment by mstrimas** [comment]
addressed in committe e1631a5fae9ae4e51083f5fa097a07967ebfc84b
